code/modules/crafting/quality_of_crafting/bombs.dm

```diff
@@ -51,7 +53,23 @@
 	attacked_atom = /obj/item/smokebomb
 	output = /obj/item/smokebomb/poison_bomb
 	craft_time = 1 SECONDS
-	craftdiff = 1
+	craftdiff = 2
+	subtypes_allowed = TRUE
+
+/datum/repeatable_crafting_recipe/bomb/napgas
+	name = "Sleeping gas Bomb"
+
+	requirements = list(
+		/obj/item/smokebomb = 1,
+		/obj/item/alch/herb/valeriana = 1,
```

Comment:
if you want it to be like valeriana sleep draught you should make it 2 valeriana for 1 mentha